server/registry/internal/storage/models/api.go

```diff
@@ -35,8 +36,8 @@ type Api struct {
 	Availability          string    // Availability of the API.
 	RecommendedVersion    string    // Recommended API version.
 	RecommendedDeployment string    // Recommended API deployment.
-	Labels                []byte    // Serialized labels.
-	Annotations           []byte    // Serialized annotations.
+	Labels                datatypes.JSON
```

Comment:
Do you know what this is using at the database level? I'm guessing/hoping json or jsonb for Postgres, but have no idea what it's doing for SQLite (bytes?).

Reply:
SQLite natively supports JSON and GORM creates this as a JSON type column.